crates/ty_python_semantic/resources/mdtest/suppressions/type_ignore.md

```diff
@@ -85,6 +85,50 @@ a = test \
   + 2  # type: ignore
 ```
 
+## Interpolated strings
```

Comment:
I didn't change the behavior for interpolated strings but I wrote this mdtest to document it, because it wasn't entirely clear to me when working on the fix.